tests/publiccloud/storage_perf.pm

```diff
@@ -84,7 +102,26 @@ sub run {
         my $output = $instance->run_ssh_command(cmd => $cmd, timeout => $runtime + 60);
         record_info('Result', $output);
 
-        # TODO: Parse results and push them to performance DB
+        $tags->{scenario} = $href->{name};
+
+        # Parse results
+        my $json = Mojo::JSON::decode_json($output);
+        $values->{read_throughput}  = $json->{jobs}[0]->{read}->{bw};
+        $values->{read_iops}        = $json->{jobs}[0]->{read}->{iops};
+        $values->{read_latency}     = $json->{jobs}[0]->{read}->{lat_ns}->{mean} / 1000;
+        $values->{write_throughput} = $json->{jobs}[0]->{write}->{bw};
+        $values->{write_iops}       = $json->{jobs}[0]->{write}->{iops};
+        $values->{write_latency}    = $json->{jobs}[0]->{write}->{lat_ns}->{mean} / 1000;
+
+        # Store values in influx-db
+        if ($url) {
```

Comment:
if we want to die when we fail to push data into DB I think we should also fail when there is no URL where to push data .

Reply:
I'd say it's ok to keep it like this in case you don't want to push the data (for whatever reason), you don't define the variable in openqa.

Reply:
ok

Reply:
I agree with jlausuch